analyzers/its/sources/ManuallyAddedNoncompliantIssues.CS/NetFramework35/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.NETFramework.ReferenceAssemblies" version="1.0.3" targetFramework="net35" developmentDependency="true" />
+  <package id="Microsoft.NETFramework.ReferenceAssemblies.net35" version="1.0.3" targetFramework="net35" developmentDependency="true" />
```

Comment:
Adding reference to the 1st one actually chooses the correct `.net` variant and adds it as a second reference